### ISSUE TITLE
Viewing description in theories

### ISSUE THREAD

**Issue by shaolintl** [normativeai/frontend#17]
Right now, when editing the description of facts in the facts base of theories, a scrolling pane is used. Once editing has finished, scrolling is no longer enabled and the additional text is not viewed. Is it possible to support it by either using a scroll down or adding a "..." button which collapse the additional text?

**Comment by lex-lex** [comment]
I cannot reproduce that. After editing, the whole description is visible, regardless of its length, isn't it? See this screenshot:
![Screenshot from 2019-04-14 18-14-48](https://user-images.githubusercontent.com/822414/56102330-374daa00-5ee1-11e9-88b9-151fa8067ad2.png)

I cannot create an example that will hide portions of the text. Or do you mean something different?

**Comment by lex-lex** [comment]
Of course, it could be a nice feature (for a clear GUI) to only show the first X characters of a description per default and only show everything on request (e.g. by clicking a "..." or "+" or similar). This could improve readibility, maybe?

**Comment by shaolintl** [comment]
Right, I dont know how I managed to make such a false conclusion. Your last comment might be useful if the text there grows too large but it is clearly of low priority. I will leave that open though and change the priority.
